administration/src/components/generation/EakGeneration.tsx

```diff
@@ -0,0 +1,92 @@
+import React from "react";
+import {EakForm} from "./EakForm";
+import {Button, ButtonGroup} from "@blueprintjs/core";
+import {CardCreationModel} from "../../models/CardCreationModel";
+import {CardType} from "../../models/CardType";
+import "./EakGeneration.css";
+import {AddEakForm} from "./AddEakForm";
+import {INTENT_DANGER} from "@blueprintjs/core/lib/esm/common/classes";
+
+interface Props {
+
+}
+
+interface State {
+    cardCreationModels: CardCreationModel[];
+}
+
+export class EakGeneration extends React.Component<Props, State> {
+
+    constructor(props: Props) {
+        super(props);
+        this.state = {
+            cardCreationModels: [EakGeneration.createEmptyCard()]
+        };
+
+        this.addForm = this.addForm.bind(this);
+        this.reset = this.reset.bind(this);
+    }
+
+    componentDidMount(): void {
+        const firstFormColumn = document.getElementsByClassName("form-column")[0];
+        const addEakBpCard = document.getElementById("add-eak-form-card")
+        if (addEakBpCard != null) {
+            addEakBpCard.style.height = firstFormColumn.clientHeight - 50 + "px"
```

Comment:
Why is that needed? Can you maybe set the height of the `form-column`s to a fixed height? That way you would not need to recalculate the height of elements

Reply:
I am not happy with this too, but as far as I can see there is no option to reach my goal otherwise. 

If I set form-column to a fixed height I fear that we maybe lose responsiveness of blueprintjs. As well, I do not want to investigate pixels by hand. 

CSS-Hacking does not help in this case, because not matter what I tried a edge case remains.

Reply:
If you really **need** to measure pixels here, use a react ref, pass the height via setState and prop to the related child element and define the style property on it.

Reply:
What do you want to achieve by setting the height to `firstFormColumn.clientHeight - 50`?